### PR TITLE
Adding truss for Qwen 7B Chat

### PR DESCRIPTION
**Pull request by htrivedi99**

### CODE REVIEW

**Review by squidarth — APPROVED** [review]
Awesome, thanks Het! One more thing for these new examples, could you start adding new ones to the ci.yaml file in the root of the repo?

So just add:

```
- model_library/qwen-7b-chat
```

to that file. It'll add the model to the models that get tested (we're not going to add every old model retroactively, but I think it would be good for new ones). Thanks!

File: model_library/qwen-7b-chat/model/model.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            return self.stream(input_ids, generation_args)
+
+        with torch.no_grad():
+            try:
+                output = self.model.generate(
+                    inputs=input_ids,
+                    **generation_args
+                )
+                return self.tokenizer.decode(output[0])
+            except Exception as exc:
```
[the remark this answers is not in the document]
Comment (htrivedi99) [reply model_library/qwen-7b-chat/model/model.py:75]: gotcha

**Comment by htrivedi99** [comment]
> Awesome, thanks Het! One more thing for these new examples, could you start adding new ones to the ci.yaml file in the root of the repo?
> 
> So just add:
> 
> ```
> - model_library/qwen-7b-chat
> ```
> 
> to that file. It'll add the model to the models that get tested (we're not going to add every old model retroactively, but I think it would be good for new ones). Thanks!

sure thing

**Comment by htrivedi99** [comment]
By the way, @squidarth I believe you or someone else from the Baseten team will have to merge these PR's, because I don't have the permission from my side.

**Pull request merged**